charts/performancetesting/values.yaml

```diff
@@ -0,0 +1,20 @@
+
+replicaCount: 1 #This is not used for now
```

Comment:
This line can be removed from the values since it's not referenced, and we're unlikely to have replicas of a cronJob instance

Reply:
Sure, I was using in previous test, for that reason it was there.